x/fees/types/params.go

```diff
@@ -0,0 +1,103 @@
+package types
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+// Parameter store key
+var (
+	DefaultFeesDenom                 = "adevi"
```

Comment:
```suggestion
	DefaultFeesDenom                 = "aevmos"
```

Reply:
I was planning to remove the denomination, since we just transfer EvmKeeper denomination.

Reply:
sounds good 👍